pom.xml

```diff
@@ -623,22 +645,12 @@
         <version>1.9.13</version>
       </dependency>
 
-      <dependency>
```

Comment:
Can you leave the jackson-databind dependency untouched. IIRC,  I had run into some interoperability issue against some version of Spark/Hive without this change. This cleanup would need more testing against different versions of Spark/Hadoop. So, better to leave this untouched.

Reply:
Sure